index.html

```diff
@@ -350,8 +350,8 @@ <h4 class="caption">Skills</h4>
         </ul>
       </div>
     </section>
-    
-    <section class="contact-form" id="contact">
+
+    <section class="contact-form1" id="contact">     
```

Comment:
For the contact form design please fix the following to match the given Figma design:

- [ ] Please add the correct background color for the contact form.
- [ ] I can see the text is overflowing from the right side so please fix that.
- [ ] Kindly fix the background image positioning and try to stick as much as possible to the Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/187448145-082e2270-9a7b-4ae7-a3da-c70184ee3493.png)  | ![image](https://user-images.githubusercontent.com/68796232/187447675-998106b3-1a2a-46ea-9e42-df04c6ef4873.png)  |